dissect/target/helpers/mount.py

```diff
@@ -32,10 +41,26 @@ def getattr(self, path: str, fh: Optional[int] = None) -> dict:
 
         try:
             st = fe.lstat()
+
             return dict(
                 (key, getattr(st, key))
-                for key in ("st_atime", "st_ctime", "st_gid", "st_mode", "st_mtime", "st_nlink", "st_size", "st_uid")
+                for key in (
+                    "st_atime",
+                    "st_ctime",
+                    # Using st_ino from the entry
+                    # However, when stating the entry through fuse(2|3)
```

Comment:
```suggestion
                    # However, when stat'ing the entry through fuse(2|3)
```
I was still confused but now I get it :smile: